recipe/bld.bat

```diff
@@ -1,8 +1,17 @@
-mkdir build
-cd build
+:: Needed so we can find stdint.h from msinttypes.
+set LIB=%LIBRARY_LIB%;%LIB%
+set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
+set INCLUDE=%LIBRARY_INC%;%INCLUDE%
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      .
+if errorlevel 1 exit 1
 
-cmake .. -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release
 nmake install
+if errorlevel 1 exit 1
 
 mkdir %SCRIPTS%
+move %LIBRARY_BIN%\xmlwf.exe %SCRIPTS%\xmlwf.exe || exit 1
```

Comment:
This line is the significant change in this PR. (We try to separate re-rendering and recipe changes in different commits.)